index.html

```diff
@@ -39,43 +41,47 @@ <h3 class="font-bold mb-2 text-xl">
             Firefox-lisäosista.
           </p>
           <p class="mb-2">
-            <a class="underline hover:bg-yellow-100" href="#">Jaksomuikkarit</a>
+            <a class="italic underline hover:text-red-400" href="#">Jaksomuikkarit</a>
           </p>
           <p><em>Player</em></p>
         </article>
 
         <h2 class="font-bold mb-4 text-xl">Menneet jaksot</h2>
 
-        <article class="-m-2 p-2 rounded hover:bg-yellow-100">
-          <a href="#">
-            <h3 class="font-bold mb-2 text-lg underline">#2 &ndash; ?</h3>
-            <footer>
-              <p class="mb-2">
-                <span class="sr-only">Julkaistu</span>
-                <time datetime="2021-03-27">27.3.2021</time>
-              </p>
-            </footer>
-            <p>Jaksokuvaus...</p>
-          </a>
-        </article>
-
-        <hr aria-hidden="true" class="my-6" />
+        <div class="flex flex-wrap">
+          <article class="w-2/4">
+            <div class="mr-3 p-3 rounded bg-white shadow-md hover:shadow-lg transition">
+              <a href="#">
+                <h3 class="font-bold mb-2 text-lg underline">#4 &ndash; ???</h3>
+                <footer>
+                  <p class="mb-2">
+                    <span class="sr-only">Julkaistu</span>
+                    <time datetime="2021-03-27">27.3.2021</time>
+                  </p>
+                </footer>
+                <p>Jaksokuvaus...</p>
+              </a>
+            </div>
+          </article>
 
-        <article class="-m-2 p-2 rounded hover:bg-yellow-100">
-          <a href="#">
-            <h3 class="font-bold mb-2 text-lg underline">#1 &ndash; Intro</h3>
-            <footer>
-              <p class="mb-2">
-                <span class="sr-only">Julkaistu</span>
-                <time datetime="2021-03-27">27.3.2021</time>
-              </p>
-            </footer>
-            <p>Jaksokuvaus...</p>
-          </a>
-        </article>
+          <article class="w-2/4">
```

Comment:
These cards get very narrow in xs. Consider making a single card take the full width in xs and using the current styles (two cards side by side) starting from md. 🧐